### ISSUE TITLE
Generate input format using .off or .obj files

### ISSUE THREAD

**Issue by hjsg1010** [charlesq34/pointnet2#66]
Can I generate .h5 train/test files using modelnet_dataset.py or data_prep_util.py ( in pointnet ) code such as dataset that can be downloaded at http://modelnet.cs.princeton.edu/ModelNet40.zip ??

I really want to test my own dataset that has vertices and edges information such as .off format.

can someone help me?

I would really appreciate it if someone reply me or give me a link with relevant code or information.

Sorry for my poor english skills.

**Comment by ljjyxz123** [comment]
Yes, you can use `provider.py` and `data_prep_util` to create your own dataset in h5 format.
@hjsg1010

**Comment by ljjyxz123** [comment]
Yes, you are right. 
Use `provider.py` and `modelnet_dataset.py` and also `modelnet_h5_dataset.py`,
`data_prep_util.py` may also help, you can copy that file from pointnet v1 repo to your pointnet v2 project. :)
@hjsg1010

**Comment by hjsg1010** [comment]
> Yes, you are right.
> Use `provider.py` and `modelnet_dataset.py` and also `modelnet_h5_dataset.py`,
> `data_prep_util.py` may also help, you can copy that file from pointnet v1 repo to your pointnet v2 project. :)
> @hjsg1010

Again, thanks for reply.
I have pointcloud data as .npy format and I can change format to .pts 
I will try it myself anyway, but if you are okay, can you teach me more specific? 
Such as use '~' function in provider.py and then use ~ in modelnet_dataset.py etc...
I know this is kind of tough request.. I would really appreciate if you can help me...
